modules/system/models/MailLayout.php

```diff
@@ -80,7 +80,6 @@ public static function findOrMakeLayout($code)
             $layout->code = $code;
             $layout->fillFromView($code);
         }
-
```

Comment:
Restore this line of whitespace please to match the other methods, plus I like it better formatted like that 😉

Reply:
that was done by mistake.